### PR TITLE
RustIntrinsic and PlatformIntrinsic are also the same ABI as Rust

### PR DESCRIPTION
**Pull request by RalfJung**
r? @eddyb

### CODE REVIEW

**Review by eddyb — APPROVED** [review]

File: src/librustc_mir/interpret/terminator.rs [hunk 1]
```diff
@@ -281,7 +281,13 @@ impl<'mir, 'tcx, M: Machine<'mir, 'tcx>> InterpCx<'mir, 'tcx, M> {
                         }
                     };
                     // Rust and RustCall are compatible
```
Comment (jonas-schievink) [inline src/librustc_mir/interpret/terminator.rs]: This is now outdated

**Comment by RalfJung** [comment]
@bors r- retry

let's fix the comment

**Pull request merged**